_component_design/table-pagination.md

```diff
@@ -0,0 +1,303 @@
+---
+title: Table Pagination
+parent: Tables
+layout: component
+category: Components
+usage: >
+  Table pagination refers to the controls that enable a user to traverse large data sets by viewing one page of table data at a time. Each page of data contains a set number of table rows (for example, 10). Users can navigate to Next and Previous pages.  
+preview-image: preview-images/tables-controls.svg
+status: stable
+resource: true
+last-modified: 2018-11-28
+helix-ui-css: false
+helix-ui-javascript: false
+---
+
+{% include toc.html %}
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Consider using pagination when a table spans multiple pages. You can also address large data sets by adding a **Load More** button that when clicked, scrolls the next group of table results up and into view.
```

Comment:
to decrease the amount of records in a table when it loads. And not 'scroll', it is 'loads' the next records below the visible records. Load more should always load the same # of records until there ain't no mo.

Reply:
done.